framework/include/dirackernels/FileConstantPointSource.h

```diff
@@ -0,0 +1,43 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+// Moose Includes
+#include "DiracKernel.h"
+
+// Forward Declarations
+class FileConstantPointSource;
+
+template <>
+InputParameters validParams<FileConstantPointSource>();
+
+/**
+ * A FileConstantPointSource DiracKernel is used to create constant point sources.
+ * Coordinates and values for the point source can be given in the input or read from file
+ */
```

Comment:
only from file

Reply:
I deleted this class since the same thing can be done with a combination of a vpp Dirackernel that reads in a csv vpp.  Their is a test for this.